repo.sh

```diff
@@ -132,10 +130,15 @@ _clone ()
             _checkout_and_update_branch
             cd ..
         else
+            if [ -n "${OPENEDX_GIT_BRANCH}" ]; then
```

Comment:
```suggestion
            if [ -n "${OPENEDX_GIT_BRANCH:-}" ]; then
```
If this is potentially unset, we should give it an empty default. (The script doesn't currently `set -u` but I would like to get it there someday.)